modules/ibi-create-config-iso.adoc

```diff
@@ -0,0 +1,316 @@
+    name: ibi-bmh <2>
+    namespace: openshift-machine-api
+  clusterDeploymentRef:
+    name: ibi-cluster-deployment <3>
+  hostname: ibi-host <4>
+  imageSetRef:
+    name: ibi-image-install <5>
+  machineNetwork: 10.0.0.0/24 <6>
+  proxy: <7>
+    httpProxy: "http://proxy.example.com:8080"
+    #httpsProxy: "http://proxy.example.com:8080"
+    #noProxy: "no_proxy.example.com"
+----
+<1> Specify the name for the `ImageClusterInstall` resource.
+<2> Specify the `BareMetalHost` resource that you want to target for the image-based installation.
+<3> Specify the name of the `ClusterDeployment` resource that you want to use for the image-based installation of the target host.
+<4> Specify the hostname for the cluster.
+<5> Specify the name of the `ClusterImageSet` resource you used to define the container release images to use for deployment.
+<6> Specify the public CIDR (Classless Inter-Domain Routing) of the external network.
+<7> Specify a proxy to use for the cluster deployment.
++
+[IMPORTANT]
+====
+If your cluster deployment requires a proxy configuration, you must do the following:
+
+* Create a seed image from a seed cluster featuring a proxy configuration. The proxy configurations do not have to match.
+* Configure the machineNetwork field in your installation manifest.
+====
+
+.. Create the `ImageClusterInstall` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f ibi-image-cluster-install.yaml
+----
+
+. Create the `ClusterDeployment` resource:
+
+.. Create a YAML file that defines the `ClusterDeployment` resource:
++
+.Example `ibi-cluster-deployment.yaml` file
+[source,yaml]
+----
+apiVersion: hive.openshift.io/v1
+kind: ClusterDeployment
+metadata:
+  name: ibi-cluster-deployment <1>
+  namespace: ibi-ns <2>
+spec:
+  baseDomain: example.com <3>
+  clusterInstallRef:
+    group: extensions.hive.openshift.io
+    kind: ImageClusterInstall 
+    name: ibi-image-install <4>
+    version: v1alpha1
+  clusterName: ibi-cluster <5>
+  platform:
+    none: {}
+  pullSecretRef:
+    name: ibi-image-pull-secret <6>
+----
+<1> Specify the name for the `ClusterDeployment` resource.
+<2> Specify the namespace for the `ClusterDeployment` resource.
+<3> Specify the base domain that the cluster should belong to.
+<4> Specify the name of the `ImageClusterInstall` in which you defined the container images to use for the image-based installation of the target host.
+<5> Specify a name for the cluster.
+<6> Specify the secret to use for pulling images from your image registry.
+
+.. Create the `ClusterDeployment` resource by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f ibi-cluster-deployment.yaml
+----
+
+. Create the `ManagedCluster` resource:
+
+.. Create a YAML file that defines the `ManagedCluster` resource:
++
+.Example `ibi-managed.yaml` file
+[source,yaml]
+----
+apiVersion: cluster.open-cluster-management.io/v1
+kind: ManagedCluster
+metadata:
+  name: sno-ibi <1>
```

Comment:
I'm not sure this would work in this example. I think this has to match the namespace where your cluster resources are created. It may be worthwhile to pick an example name more fitting that ibi-ns